yarn.lock

```diff
@@ -10876,77 +10991,92 @@ postcss@8, postcss@^7.0.14, postcss@^7.0.26, postcss@^7.0.32, postcss@^7.0.36, p
 
 prelude-ls@^1.2.1:
   version "1.2.1"
-  resolved "https://registry.yarnpkg.com/prelude-ls/-/prelude-ls-1.2.1.tgz#debc6489d7a6e6b0e7611888cec880337d316396"
+  resolved "https://registry.npmjs.org/prelude-ls/-/prelude-ls-1.2.1.tgz#debc6489d7a6e6b0e7611888cec880337d316396"
   integrity sha512-vkcDPrRZo1QZLbn5RLGPpg/WmIQ65qoWWhcGKf/b5eplkkarX0m9z8ppCat4mlOqUsWpyNuYgO3VRyrYHSzX5g==
 
 prelude-ls@~1.1.2:
   version "1.1.2"
-  resolved "https://registry.yarnpkg.com/prelude-ls/-/prelude-ls-1.1.2.tgz#21932a549f5e52ffd9a827f570e04be62a97da54"
+  resolved "https://registry.npmjs.org/prelude-ls/-/prelude-ls-1.1.2.tgz#21932a549f5e52ffd9a827f570e04be62a97da54"
   integrity sha1-IZMqVJ9eUv/ZqCf1cOBL5iqX2lQ=
 
-prettier@2.2.1, "prettier@>=2.2.1 <=2.3.0":
+prettier@2.2.1:
   version "2.2.1"
-  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.2.1.tgz#795a1a78dd52f073da0cd42b21f9c91381923ff5"
+  resolved "https://registry.npmjs.org/prettier/-/prettier-2.2.1.tgz#795a1a78dd52f073da0cd42b21f9c91381923ff5"
   integrity sha512-PqyhM2yCjg/oKkFPtTGUojv7gnZAoG80ttl45O6x2Ug/rMJw4wcc9k6aaf2hibP7BGVCCM33gZoGjyvt9mm16Q==
 
+"prettier@>=2.2.1 <=2.3.0":
+  version "2.3.0"
```

Comment:
This got messed up again so I'm seeing random formatting changes when I save in vscode

Reply:
I had a bunch of conflicts in lock file, so I have regenerated it from scratch.
What is the expected version here to be used?

Reply:
needs to be 2.2.1. see #651

Reply:
Why the heck it takes this version if we explicitly said `2.2.1` ?? Is something not working?

Reply:
you regenerated lockfile so it makes sense that it would try to get the latest compatible version. The `>=2.2.1 <=2.3.0` is coming from storybook

Reply:
This is something I am not aware about versions then
I thought, if you say `prettier: 2.2.1` the only version it is going to get for you is `2.2.1`. Now how the heck it decides, that `2.3.0` is compatible? Then why we use caret symbols at all.

Reply:
well, it has both `2.2.1` and `2.3.0` now. See the PR I linked above for more explanation and fix

Reply:
well ok..
to be honest, we should have a resolution for it instead, but reverted..

Reply:
yeah that makes sense to me. You could even sneak it into this PR

Reply:
@veekeys This got messed up again

Reply:
Added resolution...
Dare to mess up now